projects/TG-CHE200122.yaml

```diff
@@ -26,13 +26,13 @@ Sponsor:
   CampusGrid:
     Name: OSG-XSEDE
 
-ResourceAllocation:
-  XRAC:
+ResourceAllocations:
+  - Type: XRAC
     AllowedSchedds:
       - CHTC-XD-SUBMIT
     ResourceGroups:
```

Comment:
Maybe I missed the bikeshedding window on this but I find `ResourceGroups` confusing here alongside `AllowedSchedds`. I _think_ that this is differentiating between allowed access points and allowed execution points and I don't think `ResourceGroups` makes sense for that: presumably `CHTC-XD-SUBMIT` also must belong to its own resource group, no?

Reply:
I think we still have time for a bit of bikeshedding. `AllowedSchedds` are topology resources that have a "Submit Node" service; `ResourceGroups` are topology resource groups that contain the hosted CEs in front of the HPC clusters. So yeah the names are confusing.

How about `SubmitResources` and `ExecuteResourceGroups` instead? We can bikeshed over Slack if you don't like those.

Reply:
Yeah, I like that better

Reply:
Cool. @bbockelm, any objections?